src/bin/uuid.rs

```diff
@@ -0,0 +1,63 @@
+#![feature (uniform_paths)]
+
+use clap as cmd;
+use uuid::{
+    Uuid,
+};
+
+fn print_uuid (v: &str) {
+    match v {
+        // "1" => {
+        //     println! ("{}", Uuid::new_v1 () .to_hyphenated ())
+        // }
+        // "3" => {
+        //     println! ("{}", Uuid::new_v3 () .to_hyphenated ())
+        // }
+        "4" => {
+            println! ("{}", Uuid::new_v4 () .to_hyphenated ())
+        }
+        // "5" => {
+        //     println! ("{}", Uuid::new_v5 () .to_hyphenated ())
+        // }
+        _ => {
+            eprintln! ("- fail to parse -v : {}", v);
+            eprintln! ("  expecting an uuid version");
+            std::process::exit (1);
+        }
+    }
+}
+
+fn print_many_uuids (n: &str, v: &str) {
+    if let Ok (n) = n.parse::<usize> () {
+        for _ in 0 .. n {
+            print_uuid (v);
+        }
+    } else {
+        eprintln! ("- fail to parse -n : {}", n);
+        eprintln! ("  expecting an usize number");
+        std::process::exit (1);
+    }
+}
+
+fn main () {
+    let matches = cmd::App::new ("uuid")
+        .version (cmd::crate_version! ())
+        .arg (cmd::Arg::with_name ("-v")
```

Comment:
We probably don't need an argument for `-v` since there's only a single value it can take.